contracts/cw20-streams/src/contract.rs

```diff
@@ -320,8 +320,8 @@ mod tests {
         let info = mock_info("cw20", &[]);
         instantiate(deps.as_mut(), mock_env(), info.clone(), msg).unwrap();
 
-        let sender = Addr::unchecked("Alice").to_string();
-        let recipient = Addr::unchecked("Bob").to_string();
+        let sender = Addr::unchecked("alice").to_string();
+        let recipient = Addr::unchecked("bob").to_string();
```

Comment:
Those changes were unnecessary?

Reply:
It is necessary after address normalization bug fix.

Reply:
Ah cool, forgot about it. :+1: